docs/function-instructions.md

```diff
@@ -396,6 +407,7 @@ If more than five authors are present, only the top five should be returned.
 ```javascript
 mostPopularAuthors(books, authors);
```

Comment:
To be consistent with how we are naming the other functions (i.e., starting with a verb), maybe we can call this function `getMostPopularAuthors()`?